whipper/common/mbngs.py

```diff
@@ -143,8 +143,9 @@ def getName(self):
                                            i.get('artist').get('name', None)))
 
     def getIds(self):
+        # split()'s the joined string so we get a proper list of MBIDs
         return self.joiner(lambda i: i.get('artist').get('id', None),
-                           joinString=";")
+                           joinString=";").split(';')
```

Comment:
This is by far the easiest approach to do this. Otherwise the whole function needs a lot of refactoring.